src/main/deploy/pathplanner/EngageOnlyTraj.path

```diff
@@ -0,0 +1,49 @@
+{
+  "waypoints": [
+    {
+      "anchorPoint": {
+        "x": 2.2052479075196247,
+        "y": 2.66455746081135
```

Comment:
For both of these trajectories, the robot is going to try to move sideways a little bit on the way onto the charge station. You can click the position and make sure the y position is exactly the same for the two positions. 

Similar for heading, we should make the headings exactly 0 which you can do in the path planner application. They're pretty close to 0 but not quite.